terraform/modules/prom-ec2/paas-config/prometheus.conf.tpl

```diff
@@ -34,6 +34,12 @@ scrape_configs:
     file_sd_configs:
       - files: ['/etc/prometheus/targets/*.json']
         refresh_interval: 30s
+  - job_name: paas-london-targets
+    scheme: http
+    proxy_url: 'http://localhost:8080'
+    file_sd_configs:
+      - files: ['/etc/prometheus/targets/london/*.json']
+        refresh_interval: 30s
```

Comment:
can we have some `relabel_configs` here to add `region: london` to everything?

Reply:
Couldn't work out how to do this - realised you can add labels with relabel too. See next commit.